services/web-server/src/PulseEngine/index.js

```diff
@@ -175,34 +175,35 @@ export default class PulseEngine {
     });
   }
 
-  @serialize
   async innerReconcileSubscriptions() {
-    debug('Reconciling subscriptions');
+    return serialize(async () => {
```

Comment:
When I was working with @ydidwania to use decorators, they had a more complex function signature than this (taking three arguments).  But I didn't totally understand things, so I'll just ask: does this work? :)

Reply:
I was, indeed, able to start the server... I may not have used the functionality that calls this. Which annotations library did you use with yd?

Reply:
I tihnk we were using serialize.  Anyway, we can find out how it works after landing.

Reply:
We are using the legacy way of decorators. This is coming from https://github.com/taskcluster/taskcluster/blob/38c7219dc1b6f1040e64518964d2257f6500d6e8/services/web-server/.neutrinorc.js#L10. Now that we are no longer using decorators, we should perhaps remove `@babel/plugin-proposal-decorators` from the repo.